lib/chingu/basic_game_object.rb

```diff
@@ -98,40 +102,42 @@ def initialize(options = {})
     #
     def self.create(*options, &block)
       instance = self.new(*options, &block)
-      
+
       #
       # Add to parents list of game objects
       #
       instance.parent.add_game_object(instance) if instance.parent
-      
+
       #
       # Keep track of instances in class-variable
       # Used for quick access to all isntances of a certain class, for example Enemy.all
       #
```

Comment:
Probably better to remove it if it has been deactivated for so many years. I'm not sure if it works in every Ruby implementation, but this also seems to duplicate what ObjectSpace can do.
```suggestion
```